docs/src/examples.md

```diff
@@ -40,53 +40,58 @@ Also, using `DynamicQuantities.Constants`, we were able to obtain the (dimension
 Let's solve a simple projectile motion problem.
 First load the `DynamicQuantities` module:
 
-```julia
+```@example projectile
 using DynamicQuantities
 ```
 
 Set up initial conditions as quantities:
 
-```julia
+```@example projectile
 # Can explicitly import units:
 using DynamicQuantities: km, m, s, min
 
 y0 = 10km
 v0 = 250m/s
 θ = deg2rad(60)
 g = 9.81m/s^2
+nothing # hide
```

Comment:
```suggestion
y0 = 10km
v0 = 250m/s
θ = deg2rad(60)
g = 9.81m/s^2;
```

A bit simpler. Can you also use `;` rather than `nothing` elsewhere too

Reply:
hmm, no dice just yet:

![image](https://github.com/user-attachments/assets/c5d1fc03-4dee-4872-9bdb-aa4a11e69473)

I noticed you mention this in your discussion with Abhro in #175, and I've wanted to try it out ever since. Is there something else that I could try? Found some relevant discussion here: https://github.com/JuliaDocs/Documenter.jl/issues/1509